src/utils/cautContactAlerts.test.js

```diff
@@ -0,0 +1,32 @@
+import { getCautionaryAlertsType } from './cautContactAlerts'
+
+describe('getCautionaryAlertsType', () => {
+  it('combines two arrays of alerts and returns types of alerts', () => {
```

Comment:
Nitpick but would be good to include the fact that we expect the output to remove duplicate types across the two arrays in this test description - that's an important part of the functionality.